example/ionic-angular/android/app/src/main/AndroidManifest.xml

```diff
@@ -8,7 +8,9 @@
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
-        android:theme="@style/AppTheme">
+        android:theme="@style/AppTheme"
+        android:usesCleartextTraffic="true">
```

Comment:
its a sample so it does not matter but are we calling an HTTP endpoint instead of HTTPS?

Reply:
Yes, I had to add this to get the capacitor dev server to work